app/src/main/java/org/fossasia/openevent/general/social/SocialLinksFragment.kt

```diff
@@ -80,7 +80,15 @@ class SocialLinksFragment : Fragment() {
                 Snackbar.make(socialLinksCoordinatorLayout, it, Snackbar.LENGTH_LONG).show()
             })
 
-        loadSocialLink()
+        socialLinksViewModel.mutableInternetError
```

Comment:
Subscribe to `internetError`, make `mutableInternetError` private

Reply:
Sure, what's the benefit btw?

Reply:
So that contributors don't make the mistake of posting values from fragments and activities

Reply:
Oh Okay..

Reply:
MVVM View and ViewModel don't have a bidirectional relationship. VM posts and view listens. View only interacts with VM by asking for things. It never provides something to VM